TTCN3_Project/CANFilter/com/testingtech/playits/canfilter/tests/CANFilterTest.java

```diff
@@ -0,0 +1,48 @@
+package com.testingtech.playits.canfilter.tests;
+
+import java.util.Arrays;
+import java.util.Date;
+import java.util.Hashtable;
+import java.util.List;
+
+import org.junit.runner.RunWith;
+import org.junit.runners.Parameterized;
+import org.junit.runners.Parameterized.Parameters;
+
+import com.testingtech.playits.canfilter.Car2XEntry;
+
+@RunWith(Parameterized.class)
+public class CANFilterTest {
+
+	private Hashtable<String, Car2XEntry> car2xValues = new Hashtable<String, Car2XEntry>();
+	private int duration;
+	private int expectedTimeouts;
+	private long startTime;
+	private long currentTime;
+
+	public void setUp() {
+		car2xValues.put("vehicle_speed", new Car2XEntry());
```

Comment:
These  strings should be excluded to a constants class to provide better maintainability.